htmresearch/frameworks/nlp/classification_model.py

```diff
@@ -371,3 +383,103 @@ def classifyText(self, token, reset=0):
                            implies no decision could be made.
     """
     raise NotImplementedError
+
+
+  def save(self, saveModelDir):
+    """ Save the model in the given directory via pickling.
```

Comment:
> Please add that the saved model will be named "model.pkl".

That's an internal implementation detail - I don't think that should be in the user facing comments.  From the user's standpoint the saved model is the entire directory.